index.js

```diff
@@ -8,8 +8,51 @@ const http = require('http'),
         The time is: ${rightNow}
       </p>
     </body>
+ 
+      <body>
+   <div style="position:  absolute; bottom: 5px; background-color: white">
+<h2> Characters in The Lord of The Rings </h2>
+ <ul>
+   <li>   <b> Frodo Baggins </b>  </li>
+   <li>   <b> Bilbo Baggins </b>  </li>
+   <li>   <b> Gandalf '<i> (the Grey, later the White) </i>' </b> </li>
+   <li>  <b> Samwise Gamgee </b>  </li>
+   <li>  <b> Aragorn
+  <i> Ranger of the North
+  </i> </b>  </li>
+   <li>  <b> Arewn </b>  </li>
+   <li> <b> Elrond </b>  </li>
+   <li>   <b> Galadriel </b>  </li>
+   <li>   <b> Legolas </b>  </li>
+<li>  <b> Bro-omir </b>  </li>
+<li>  <b> Gimli  </b> <i> The one, the only boss-man RHYS DAVIES </i> </li>
+</ul>
+<p> </p>
```

Comment:
Remove unneeded `<p>` block. I am sure you added this for spacing but we will correct that in a more semantic way in the future. It will look less then ideal for now.